services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttaja.kt

```diff
@@ -10,6 +11,20 @@ import java.util.UUID
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.stereotype.Repository
 
+@Schema(description = "Api response of user data of given Hanke")
+data class HankeKayttajaResponse(
+    @field:Schema(description = "Hanke users") val kayttajat: List<HankeKayttajaDto>
+)
+
+@Schema(description = "Hanke user")
+data class HankeKayttajaDto(
+    @field:Schema(description = "Id, set by the service") val id: UUID,
+    @field:Schema(description = "Email address") val sahkoposti: String,
+    @field:Schema(description = "Full name") val nimi: String,
+    @field:Schema(description = "Role in Hanke, null for Hanke creator") val rooli: Role?,
```

Comment:
I think we're missing a task to create a HankeKayttaja for the creator of a hanke.

Reply:
The creator is not returned at all currently, since we're listing HankeKayttaja and that is not created for the creator.

Reply:
So should the description be just "role in hanke"?

Reply:
Yes, I don't think there should be anything special with the creator.